### PR TITLE
Ocd-1799

### PR DESCRIPTION
**Pull request by brianlindsy**

### CODE REVIEW

File: src/app/collections/collection.directive.js [hunk 1]
```diff
@@ -31,9 +31,10 @@
     }
 
     /** @ngInject */
-    function CollectionController ($filter, $localStorage, $log, $timeout, RELOAD_TIMEOUT, collectionsService, networkService) {
+    function CollectionController ($filter, $localStorage, $log, $timeout, API, authService, collectionsService, networkService, RELOAD_TIMEOUT) {
```
Comment (andlar) [inline src/app/collections/collection.directive.js]: Linting should give an error about non-alphabetized parameters on this line

File: src/app/collections/collection.directive.js [hunk 2]
```diff
@@ -31,9 +31,10 @@
...
         var vm = this;
 
+        vm.getCsv = networkService.getSedCsv;
```
Comment (andlar) [inline src/app/collections/collection.directive.js]: Pretty sure this isn't needed

File: src/app/collections/collection.html [hunk 3]
```diff
@@ -125,6 +124,11 @@
           <a href ng-click="vm.triggerClearFilters()" ng-if="vm.isCategoryChanged()">Clear Filters</a>
         </div>
       </div> <!-- #filters -->
+      <div class="form group" ng-if="vm.sedDownloadAllButton()">
```
Comment (andlar) [inline src/app/collections/collection.html]: This whole section, including the stuff in the controller, should be in the `collections/sed/`component somewhere. The stuff that goes in the collections directive is supposed to be stuff that could be used by multiple collections, but a button to download SED details isn't like that

File: src/app/collections/collection.directive.js (hunk 1 above)
Comment (brianlindsy) [reply src/app/collections/collection.directive.js]: Im confused about this when i saw it in the linter output...isnt this is alpha order?

Comment (andlar) [reply src/app/collections/collection.directive.js]: Capitals come before lower case in the rule-set

File: src/app/collections/sed/sed.html [hunk 4]
```diff
@@ -1,8 +1,11 @@
 <ai-collection call-function="vm.viewDetails(id)" collection-key="sed" columns="vm.columnSet" filters="vm.filters" refine-model="vm.refineModel">
   <ai-body-text>
-    <p>
+    <div>
       This list includes all health IT products that have been certified with Safety Enhanced Design (SED):
-    </p>
+      <div class="btn-group">
+        <a type="button" class="btn btn-sm btn-ai-success" href="{{ vm.API }}/certified_products/sed_details?api_key={{ vm.API_KEY }}"><i class="fa fa-cloud-download"></i><span class="sr-only">Download All SED Details</span>Download All SED Details</a>
+      </div>
```
Comment (andlar) [inline src/app/collections/sed/sed.html]: A screen-reader reading this would have "Download All SED Details" read out loud twice. The `class="sr-only"` is used to make text only parsed by screen-readers, but not shown visibly.

File: src/app/collections/collection.directive.js [hunk 5]
```diff
@@ -45,7 +45,6 @@
         vm.triggerClearFilters = triggerClearFilters;
         //vm.triggerRestoreState = triggerRestoreState;
         vm.triggerSearch = triggerSearch;
-
         activate();
```
Comment (andlar) [inline src/app/collections/collection.directive.js]: All of the other controllers have a space between the list of "public" methods and the "activate()" call

File: src/app/collections/collection.html [hunk 6]
```diff
@@ -23,7 +23,6 @@
             <label for="generalFilter"><span class="sr-only">{{ vm.searchText }}</span></label>
           </div>
         </div>
-
         <div class="col-sm-8" ng-if="vm.filters.length > 0">
```
Comment (andlar) [inline src/app/collections/collection.html]: Why was this removed?
